### ISSUE TITLE
How to use AnchorLoss in Pose estimation

### ISSUE THREAD

**Issue by YuQi9797** [slryou41/AnchorLoss#8]
thanks for your sharing.
Could you tell me how to apply the anchor loss in pose estimation?

My input is heatmap, it's shape : N, C, H, W.  and my target also is N, C, H, W.

And, how to apply the anchor loss in pose estimation?

looking forward your reply!

**Comment by slryou41** [comment]
Hi, thanks for your interest! Could you tell me more about the target? You have to design your anchor probability based on your target distribution. Are you using the gaussian heatmap?

**Comment by YuQi9797** [comment]
Yeah, I'm using a Gaussian heatmap. However, my output channel is NUM_joints +1. One channel is used to output a heatmap of all key points of the target. The loss function I use is MSE.

If I want to use Anchor Loss, I should only feed the input is (N,num_joints, H, W). But I don't know how to apply the code you shared to my project. Could you share the Anchor Loss code for pose estimation?
